src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@shinfan @bjwatson This looks like a discrepancy between Python and Java. Python sends "gapic/0.15.1", where 0.15.1 is the version of the GAPIC client library, but Java seems to be sending the version of GAPIC generator itself?